server/lib/notifications.js

```diff
@@ -215,9 +215,62 @@ const notifyUpdateSubscribers = async activity => {
   const emailOpts = { from: activity.data.fromEmail };
   const update = await models.Update.findByPk(activity.data.update.id);
   const allUsers = await update.getUsersToNotify();
-  return notifySubscribers(allUsers, activity, emailOpts);
+  const modifiedActivity = replaceVideosByImagePreviews(activity);
+  return notifySubscribers(allUsers, modifiedActivity, emailOpts);
 };
 
+function replaceVideosByImagePreviews(activity) {
+  const iframePreviewRegex = /<iframe\s[^>]*src="([^"]+)"[^>]*><\/iframe>/gi;
+  activity.data.update.html = activity.data.update.html
+    .replace(iframePreviewRegex, (_, href) => {
+      const { service, id } = parseServiceLink(href);
+      const imgSrc = constructPreviewImageURL(service, id);
+      if (imgSrc) {
+        return `<img src="${imgSrc}" alt="${service} content">`;
+      } else {
+        return '';
+      }
+    })
```

Comment:
![image](https://user-images.githubusercontent.com/1556356/119616640-203c4e80-be01-11eb-9116-d24b7a7b62ee.png)


Regarding this warning, I would add a small line to remove all iframes that we haven't been able to convert

```suggestion
    }).replace(/<iframe\s[^>]><\/iframe>/gi, '')
```

Reply:
We can also use `sanitize-html` for this

Reply:
Done. 👍🏽 Didn't notice this warning in CI. 🤔